inc/modules/export/class-table.php

```diff
@@ -0,0 +1,451 @@
+	 */
+	public function single_row( $item ) {
+		$format = $this->getTinyHash( $item['format'] );
+		echo "<tr data-id='{$item['ID']}' data-format='{$format}'>";
+		$this->single_row_columns( $item );
+		echo '</tr>';
+	}
+
+	/**
+	 * This method is called when the parent class can't find a method
+	 * for a given column. For example, if the class needs to process a column
+	 * named 'title', it would first see if a method named $this->column_title()
+	 * exists. If it doesn't this one will be used.
+	 *
+	 * @see WP_List_Table::single_row_columns()
+	 *
+	 * @param object $item A singular item (one full row's worth of data)
+	 * @param string $column_name The name/slug of the column to be processed
+	 *
+	 * @return string Text or HTML to be placed inside the column <td>
+	 */
+	public function column_default( $item, $column_name ) {
+		return esc_html( $item[ $column_name ] );
+	}
+
+	/**
+	 * @param array $item
+	 *
+	 * @return string
+	 */
+	public function column_cb( $item ) {
+		return sprintf( '<input type="checkbox" name="ID[]" value="%s" />', $item['ID'] );
+	}
+
+	/**
+	 * @param array $item A singular item (one full row's worth of data)
+	 *
+	 * @return string Text to be placed inside the column <td>
+	 */
+	public function column_file( $item ) {
+		$html = '<div class="export-file">';
+		$html .= $this->getIcon( $item['file'] );
+		$html .= '<div class="export-file-name">' . esc_html( $item['file'] ) . '</div>';
+		$html .= '</div>';
+
+		$delete_url = sprintf( '/admin.php?page=%s&action=%s&ID=%s', $_REQUEST['page'], 'delete', $item['ID'] );
+		$delete_url = get_admin_url( get_current_blog_id(), $delete_url );
+		$delete_url = esc_url( add_query_arg( '_wpnonce', wp_create_nonce( 'bulk-files' ), $delete_url ) );
+		$onclick = 'onclick="if ( !confirm(\'' . esc_attr( __( 'Are you sure you want to delete this?', 'pressbooks' ) ) . '\') ) { return false }"';
+		$actions['delete'] = sprintf(
+			'<a href="%s" aria-label="%s" ' . $onclick . '>%s</a>',
+			$delete_url,
+			/* translators: %s: filename */
+			esc_attr( sprintf( __( 'Delete &#8220;%s&#8221;' ), $item['file'] ) ),
+			_x( 'Delete', 'verb' )
+		);
+
+		$download_url = get_admin_url( get_current_blog_id(), "/admin.php?page=pb_export&download_export_file={$item['file']}" );
+		$actions['download'] = sprintf(
+			'<a href="%s" aria-label="%s" >%s</a>',
+			$download_url,
+			/* translators: %s: filename */
+			esc_attr( sprintf( __( 'Download &#8220;%s&#8221;' ), $item['file'] ) ),
+			_x( 'Download', 'verb' )
+		);
+
+		$html .= $this->row_actions( $actions );
+
+		return $html;
+	}
+
+	/**
+	 * @param array $item A singular item (one full row's worth of data)
+	 *
+	 * @return string Text to be placed inside the column <td>
+	 */
+	public function column_pin( $item ) {
+		$format = $this->getTinyHash( $item['format'] );
+		$html = "<input type='checkbox' name='pin[{$item['ID']}]' value='{$format}' " . checked( $item['pin'], true, false ) . '/>';
+		return $html;
```

Comment:
Each of these needs a text label for accessibility. This means they need an ID attribute and a `<label for="<ID>">Pin this file</label>`. To visually hide the label in mobile view (when the column header is shown next to the checkbox), you can use this SCSS mixin:

https://github.com/pressbooks/pressbooks/blob/dev/assets/src/styles/organize.scss#L89-L96